### PR TITLE
Added ReadMe first draft

### PR DESCRIPTION
**Pull request by geoffhull03**
Added the first draft of the ReadMe with some basic instructions on how the app works and how it can be used in this branch. The app works as planned; users can obtain an access token, hit the Get Me endpoint, and refresh their access token successfully.

### CODE REVIEW

**Comment by colinfujiiprocore** [comment]
I removed the logs/temp files from the PR and added rules to the gitignore to ignore those files from now on. You can remove files like that using 'git rm [filepath]'

File: README.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-
-* Deployment instructions
-
-* ...
+### Setup Instructions
+This following steps are instructions to launch and view a simple Ruby on Rails applicaiton that authenticates with Procore's API using the OAuth 2.0 Authorization Code Grant Type flow. The application is configured to access Procore's environment. If you do not have access to this environment or if you're working out of Procore's developer sandbox environment, all root URLs in the `login_controller` and `users_controller` must be re-configured to https://sandbox.procore.com.
+
+1. Clone this repository
+2. Run `bundle install`
+3. The Gemfile includes the Figaro gem. This gem should automatically create an untracked file: `config/application.yml`
```
Comment (colinfujiiprocore) [inline README.md]: Figaro won't automatically create this file unless the user runs 'bundle exec figaro install'. Just 'bundle install' won't create the file, so they'll need to run both.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+### Setup Instructions
+This following steps are instructions to launch and view a simple Ruby on Rails applicaiton that authenticates with Procore's API using the OAuth 2.0 Authorization Code Grant Type flow. The application is configured to access Procore's environment. If you do not have access to this environment or if you're working out of Procore's developer sandbox environment, all root URLs in the `login_controller` and `users_controller` must be re-configured to https://sandbox.procore.com.
```
Comment (colinfujiiprocore) [inline README.md]: We should make it so the developer doesn't need to go into the controllers to do this. Better to have the environment's base URL be in the application.yml file that they're already going to need to modify for this example app.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
+        * CLIENT_ID: ''
+        CLIENT_SECRET: ''
+        REDIRECT_URL: 'http://localhost:3000/login/callback'
+
+    * Client ID and Client Secret values are provided when [creating an application](https://developers.procore.com/documentation/new-application) in the Procore Developer Portal. The redirect URI above should be added to your application, which can be done on your application's home page.
+    * After these values have been configred within the `application.yml` file, make sure to save your changes.
+4. Navigate to the directory where the repository was cloned to and launch your Rails server
+5. The landing page will include a button that says, "Sign In to Procore". Click this button and enter your Procore email address/password.
+6. After authenticating with Procore, you will be redirected back to the sample application. This page will include a table containing the first five and last five characters of your access token and your refresh token. In addition, there will be timestamps correspoding to when the access token was generated and when it expires (2 hours after generation).
```
Comment (nick-morrill) [inline README.md]: >`This page will include a table containing the first five and last five characters of your access token and your refresh token.`

IMO this should be:
>`This page will include a table containing the first and last five characters of both your access token and your refresh token.`

File: README.md (hunk 1 above)
Comment (geoffhull03) [reply README.md]: Updated this portion to reflect the need to run the `bundle exec figaro install` command in the most recent commit

File: README.md (hunk 2 above)
Comment (geoffhull03) [reply README.md]: Reconfigured controllers for environment flexibility and updated the README with new instructions on setup.

File: README.md (hunk 3 above)
Comment (geoffhull03) [reply README.md]: Made this change to the README!

**Pull request merged**